cmd/dev/ci/monorepo/main.go

```
@@ -5,28 +5,47 @@ package monorepo
-}
+// Package-level scratch variables read by the helpers in this package. They are
+// populated from per-command local flag values in PersistentPreRun (see
+// NewCommand) and the leaves' RunE so that constructing the command tree
+// concurrently does not race on flag default writes.
+var (
+	rootDirectory string
+	verbose       bool
+	debug         bool
+	pr            string
+	revisionRange string
+	branch        string
```

golangci-lint (2.12.2)

[error] var branch is unused

(unused)